backend/backend/settings.py

```diff
@@ -140,3 +141,14 @@
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
 SECRET_KEY = get_random_secret_key()
+
+REST_FRAMEWORK = {
+    "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
+}
+
+SPECTACULAR_SETTINGS = {
+    "TITLE": "Activist.org API",
+    "DESCRIPTION": "Open-source, nonprofit activism platform",
+    "VERSION": "1.0.0",
```

Comment:
Maybe do `0.1.0` for now as we're working towards beta? We should have the API at the same version as everything else for now until we break it up? Or are we doing different versions for the backend and frontnend?